### ISSUE TITLE
Polymelia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6809]
**Preferred term label:**
Polymelia

**Synonyms**
Accessory limb

**Definition (free text, please give PubMed ID)**
Polymelia is a congenital anomaly, which is defined as the presence of accessory limbs attached to various body regions and could be classified as cephalomelia (extra-limb attached to the head), notomelia (extra-limb attached to the back bone), thoracomelia (extra-limb attached to the thorax), and pyromelia (extra-limb attached to the pelvis). 
PMID:23814690

**Parent term (use hpo.jax.org/app)**
Abnormality of limbs HP:0040064

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
